### DISCUSSION TITLE
Val in CatBoost's plot_tree

### DISCUSSION THREAD

**Discussion by ManRev** [catboost/catboost#2623] [General]
Could you please help me understand the meaning of val = v ([see example here](https://i.stack.imgur.com/yls98.png) from [the documentation](https://catboost.ai/en/docs/concepts/python-reference_catboost_plot_tree)) in the leaves of the tree plotted with model.plot_tree in the CatBoost library?

I cannot parse the [documentation](https://catboost.ai/en/docs/concepts/output-data_model-value-output) on that subject. In [a previous answer](https://stackoverflow.com/questions/63407613/catboost-plot-tree-understanding), it was said that val "are values predicted by the tree." How can the values be numbers between -0.5 and 0.5 when the task is to rank or classify binary entries?

**Comment by ek-ak** [comment]
Hello! 
For any document the result from applying the model  is (the sum of the corresponding leave values in every tree) * model_scale + model_bias. This is RawFormulaVal.

RawFormulaVal  is a number resulting from applying the model.
Probability  is a number indicating the probability that the object belongs to the class (a sigmoid of the result of applying the model).
Class — the predicted class (output with the value 1 if the probability is higher than 0.5, otherwise 0).
